audio/portaudio/Portfile

```diff
@@ -3,8 +3,10 @@
 PortSystem          1.0
 
 name                portaudio
-version             19.20140130
-revision            2
+set major			19
+set minor			6
+set reldate			20161030
```

Comment:
Some nitpicking: ideally use spaces instead of tabs :)
Same comment for a few more places in the `Portfile`.

Reply:
Fixed that; also squashed and rebased.

Reply:
Changed, squashed and rebased.

Reply:
Fine, but I'm not pulling this when it breaks the version number. As it stands right now, the version number of the new port will be considered older than that of the old port. I've commented on this above repeatedly.